File: app/src/main/java/com/magnitudestudios/GameFace/Bases/BaseActivity.java
package com.magnitudestudios.GameFace.Bases;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.magnitudestudios.GameFace.GameFace;
import com.magnitudestudios.GameFace.Network.FirebaseHelper;

public class BaseActivity extends AppCompatActivity {
    public FirebaseHelper firebaseHelper;
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        firebaseHelper = ((GameFace) getApplicationContext()).firebaseHelper;
    }
}

File: app/src/main/java/com/magnitudestudios/GameFace/Bases/BaseActivity.kt
package com.magnitudestudios.GameFace.Bases

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.magnitudestudios.GameFace.GameFace
import com.magnitudestudios.GameFace.Network.FirebaseHelper

open class BaseActivity : AppCompatActivity() {
    @JvmField
    var firebaseHelper: FirebaseHelper? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        firebaseHelper = (applicationContext as GameFace).firebaseHelper
    }
}